packages/mobile/src/fiatExchanges/reducer.ts

```diff
@@ -5,38 +5,38 @@ import i18n from 'src/i18n'
 import { getRehydratePayload, REHYDRATE } from 'src/redux/persist-helper'
 import { RootState } from 'src/redux/reducers'
 
-export enum CiCoProvider {
-  Moonpay = 'Moonpay',
-  Ramp = 'Ramp',
-  Simplex = 'Simplex',
-  Transak = 'Transak',
+export enum CicoProviderNames {
```

Comment:
Is there a reason for this change? The previous implementation was following the style guidelines for our enums: https://celo-org.slack.com/archives/CL6JFBALE/p1589448552015000

Reply:
Yeah good point. I forgot we had a style guide now and just reverted to how I was used to doing them. Thanks for flagging I will revert these changes.